proto/cosmos/group/v1/state.proto

```diff
@@ -0,0 +1,198 @@
+syntax = "proto3";
+
+package cosmos.group.v1;
```

Comment:
We should probably put this in a separate package, i.e. `cosmos.group.state.v1`, because state breakage happens at a different cadence from API breakage and we don't want to tightly couple those.

Reply:
The issue when doing that is we get import cycles in the generated code.
Or we need to add non state related types in the state.proto.

Reply:
What are the specific import cycles?

Reply:
between the types and state.
Query imports state and state imports the types `ProposalStatus`, `TallyResult`, that imho has nothing to do in state.

Reply:
Then maybe my comment doesn't make sense. If we need to make major changes to state that can be state.v2 but maybe keeping it all together for now is simpler